iothub/device/src/Edge/InstalledCertificateValidator.cs

```diff
@@ -32,10 +32,13 @@ public static InstalledCertificateValidator Create(IList<X509Certificate2> certs
         private void SetupCertificateValidation()
         {
             Debug.WriteLine("InstalledCertificateValidator.SetupCertificateValidation()");
-
-            foreach (var cert in this._certs)
+#if NET451
+            var store = new X509Store(StoreName.Root, StoreLocation.CurrentUser);
+#else
+            using var store = new X509Store(StoreName.Root, StoreLocation.CurrentUser);
+#endif
+            foreach (X509Certificate2 cert in _certs)
```

Comment:
What do you think we intend to do here? The `store` looks to be a local `X509Store` variable, that we don't reference anywhere. Do we simply want to validate that the certificates can be added to **_any_** store? Do we think it's relevant if we add them to a new store or to the same store?

Reply:
I think this just installs certificates in the cert store , it's the current user's root store btw
https://docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.x509certificates.x509store?view=netframework-4.7.2